sdk/identity/identity/src/constants.ts

```diff
@@ -58,9 +58,17 @@ export const ALL_TENANTS: string[] = ["*"];
 /**
  * @internal
  */
-export const CACHE_CAE_SUFFIX = ".cae";
+export const CACHE_CAE_SUFFIX = "cae";
 
 /**
  * @internal
  */
-export const CACHE_NON_CAE_SUFFIX = ".nocae";
+export const CACHE_NON_CAE_SUFFIX = "nocae";
+
+/**
+ * @internal
+ *
+ * The default name for the cache persistence plugin.
+ * Matches the constant defined in the cache persistence package.
+ */
+export const DEFAULT_TOKEN_CACHE_NAME = "msal.cache";
```

Comment:
Matches https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/identity/identity-cache-persistence/src/platforms.ts#L34